controllers/volumereplicationgroup_controller.go

```diff
@@ -538,65 +538,23 @@ const (
 	MModesLabel = "ramendr.openshift.io/maintenancemodes"
 )
 
-func (v *VRGInstance) processVRG() (ctrl.Result, error) {
+func (v *VRGInstance) processVRG() ctrl.Result {
 	if err := v.validateVRGState(); err != nil {
-		// record the event
-		v.log.Error(err, "Failed to validate the spec state")
```

Comment:
Previously we logged an error and now log info level message. This seems like bad change, and mixing it with big refactoring make it hard to detect.

Reply:
A couple years ago we discussed the difference between Info and Error log entries and Info was preferred as it does not clutter the log with stack trace when it is not needed.